docs/index.md

```diff
@@ -1827,6 +1827,10 @@ suite('Array', function () {
 
 ### Exports
 
+The `exports` interface allows for organizing tests in a modular fashion. It is particularly useful in larger projects where test suites can be segmented into different files.
```

Comment:
[Style] Heh, looks like we have a bit of a discrepency: BDD, TDD, and (previously Exports were in **BoldPascalCased**; QUnit and `require` were not. Let's not change in this PR:

```suggestion
The **Exports** interface allows for organizing tests in a modular fashion. It is particularly useful in larger projects where test suites can be segmented into different files.
```